dora/stress/shell/src/main/java/alluxio/stress/cli/worker/StressWorkerBench.java

```diff
@@ -415,6 +396,10 @@ private BenchThread(BenchContext context, int targetFileIndex, FileSystem fs) {
       mResult.setParameters(mParameters);
       mResult.setBaseParameters(mBaseParameters);
       mIsRandomRead = mParameters.mIsRandom;
+      mRandom = new Random(mParameters.mRandomSeed);
```

Comment:
if you are using the same random seed for all threads, then wouldn't the offset and length sequence of all threads be the same as well?

Reply:
I understand your motivation now. The threads were reading from one single offset and with the same length throughout the benchmark, and this PR makes them read with different offsets and lengths. Makes sense to me.

Reply:
yep, i used to worry the same seed problem as well, but i think as all threads read enough times it won't be matter.